blocks/library/latest-posts/style.scss

```diff
@@ -13,32 +13,22 @@
 
 		li {
 			margin: 0 16px 16px 0;
+			width: 100%;
 		}
 	}
 
-	&.columns-1 li {
```

Comment:
Was it intentional to drop the `columns-1` styles? I guess it would make sense, since we wouldn't want the single column width to be constrained.

Reply:
Yes, it was intentional. One column layout would be default `width: auto`. We'll come back to columns grid later on. But I think it doesn't hurt to start basics like this.

Reply:
The slider control doesn't allow the value of `columns` to be set to 6. Should we also limit the loop to stop at 5?

Reply:
Yeah, I was about to ask about slider setting. If it's not filterable it makes sense to stop loop at 5.

Reply:
Hmm I can still see that you can set `columns` to be set to 6. I'm leaving CSS as it is for now.

All the blocks that have grid view should have same or similar grid CSS (grid gap, margins) though. We can discuss it later on, do we use CSS Grid etc.